src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -515,16 +515,16 @@ public int Space_WithinSquares
             set { SetBooleanOption(CSharpFormattingOptions.SpaceWithinSquareBrackets, value); }
         }
 
-        public int Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
+        public string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
```

Comment:
And if it was already shipped, perhaps we're OK saying the definition changes, but we just need to accept setting of the older values so we don't break .vssettings files or profiles or upgrade scenarios.